install/0.2.0/kfserving.yaml

```diff
@@ -209,7 +209,7 @@ spec:
                       - storageUri
                       type: object
                     tensorrt:
-                      description: Spec for TensorRT Inference Server (https://github.com/NVIDIA/triton-inference-server)
+                      description: Spec for TensorRT Inference Server (https://github.com/triton-inference-server/server)
```

Comment:
Let's not modify the released yaml.

Reply:
Done. Removed changes from all released yamls.